core/src/main/scala/org/apache/spark/deploy/DeployMessage.scala

```diff
@@ -77,10 +77,10 @@ private[deploy] object DeployMessages {
   object DecommissionWorker extends DeployMessage
 
   /**
-   * A message that sent by the Worker to itself when it receives PWR signal,
+   * A message that sent by the Worker to itself when it receives a signal,
    * indicating the Worker starts to decommission.
    */
-  object WorkerSigPWRReceived extends DeployMessage
+  object WorkerDecomSigReceived extends DeployMessage
```

Comment:
Shall we use a full word `Decommission` instead of `Decom`? e.g. `WorkerDecommissionSigalReceived`?

Reply:
Sure.